fretwell-distributed-government.md

```diff
@@ -1,98 +1,93 @@
 # Distributed government is the path to a truly open, empathetic, inclusive civil society
 Luke Fretwell
 
-Society has reached a point in the organizational and civic evolutionary cycle where [distributed government teams](https://distributedgov.com/) will play an undeniable and critical role in the future of highly effective, empathetic and inclusive public services.
+Society has reached a point in the organizational and civic evolutionary cycle where [distributed government teams](https://distributedgov.com/) will play an undeniable and critical role in the future of highly effective, empathetic, and inclusive public services.
```

Comment:
Just to clarify: Are we talking about the U.S. here? How generalizable to other contexts (if at all) are your statistics and arguments?